roles/migrationcontroller/templates/controller.yml.j2

```diff
@@ -61,6 +61,8 @@ spec:
           value: webhook-server-secret
         - name: MIGRATION_REGISTRY_IMAGE
           value: {{ migration_registry_image_fqin }}
+        - name: ENABLE_INTELLIGENT_PV_RESIZE
```

Comment:
Please use configmap instead of Pod environment variables for consistency

Reply:
Done